api/v1alpha1/virtualmachineimage_conversion.go

```diff
@@ -226,13 +228,29 @@ func (dst *VirtualMachineImageList) ConvertFrom(srcRaw conversion.Hub) error {
 // ConvertTo converts this ClusterVirtualMachineImage to the Hub version.
 func (src *ClusterVirtualMachineImage) ConvertTo(dstRaw conversion.Hub) error {
 	dst := dstRaw.(*v1alpha2.ClusterVirtualMachineImage)
-	return Convert_v1alpha1_ClusterVirtualMachineImage_To_v1alpha2_ClusterVirtualMachineImage(src, dst, nil)
+	if err := Convert_v1alpha1_ClusterVirtualMachineImage_To_v1alpha2_ClusterVirtualMachineImage(src, dst, nil); err != nil {
```

Comment:
The public ones are ones that conversion-gen was able to automatically generate. The private ones that otherwise follow the same naming convention are ones we have written to fill in the missing pieces. I think I cribbed this convention from CAPI.